packages/frontend/src/app/pages/auth/auth.page.html

```diff
@@ -76,6 +77,7 @@ <h2>{{ 'pages.auth.welcome.register' | translate }}</h2>
               [(ngModel)]="confirmPassword"
```

Comment:
[non-blocking] Would love for this input to use ion-input rather than this manually-styled input.